pkg/cmd/pipeline/start.go

```diff
@@ -97,49 +100,145 @@ tkn pipeline start foo --param NAME=VALUE --resource source=scaffold-git  -s Ser
 	c.Flags().StringSliceVarP(&res, "resource", "r", []string{}, "pass the resource name and ref")
 	c.Flags().StringSliceVarP(&params, "param", "p", []string{}, "pass the param")
 	c.Flags().StringVarP(&svc, "serviceaccount", "s", svc, "pass the serviceaccount name")
-
+	c.Flags().BoolVarP(&last, "last", "l", false, "re-run the pipeline using last pipelinerun values")
 	return c
 }
 
 func startPipeline(out io.Writer, opt startOptions, p cli.Params, pName string) error {
+	pr := &v1alpha1.PipelineRun{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace:    p.Namespace(),
+			GenerateName: pName + "-run-",
```

Comment:
nit: I wonder if we need to append `-run` to the `PipelineRun` name :thinking:

Reply:
Then it will be like Pipeline-<randomstring>.